addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -178,15 +178,9 @@ publicWidget.registry.ProductWishlist = publicWidget.Widget.extend(VariantMixin,
                 add_qty: parseInt(qty_id, 10),
                 display: false,
             },
-        }).then(function (resp) {
-            if (resp.warning) {
-                if (! $('#data_warning').length) {
-                    $('.wishlist-section').prepend('<div class="mt16 alert alert-danger alert-dismissable" role="alert" id="data_warning"></div>');
-                }
-                var cart_alert = $('.wishlist-section').parent().find('#data_warning');
-                cart_alert.html('<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button> ' + resp.warning);
-            }
-            $('.my_cart_quantity').html(resp.cart_quantity || '<i class="fa fa-warning" /> ');
```

Comment:
no more fallback if qty is 0

Add as first product in your cart a product of out stock from your wishlist, you will have a glitch:
![image](https://user-images.githubusercontent.com/4848797/178286902-56dc60b8-905f-4c17-ad0b-50b62057ba70.png)

Reply:
Interesting. Seems like that's also an issue (in 13 itself) for the "recently viewed flow" you talk about above, so that's a good thing to fix in the "unified" version.

Though it looks like the warning thing is not complete, it doesn't show up from the page of the recently viewed widget (it also immediately disappears when adding the only item from the wishlist but I don't think I can do much about that...)

Reply:
> (it also immediately disappears when adding the only item from the wishlist but I don't think I can do much about that...

Yes spotted it yesterday too :/ less critical since user is redirected to cart directly after and will see the fucked qty, but not perfect, i'm agree with you.